drivers/windows/windows_store.go

```diff
@@ -261,7 +261,7 @@ func (ep *hnsEndpoint) UnmarshalJSON(b []byte) error {
 	}
 	if v, ok := epMap["Addr"]; ok {
 		if ep.addr, err = types.ParseCIDR(v.(string)); err != nil {
-			return types.InternalErrorf("failed to decode endpoint IPv4 address (%s) after json unmarshal: %v", v.(string), err)
+			logrus.Warnf("failed to decode endpoint IPv4 address (%s) after json unmarshal: %v", v.(string), err)
```

Comment:
We are changing INternalErrorf to warnf.  Instead of using Warnf , should we use Errorf?

Reply:
I guess this is a judgment call. It seemed more like a warning to me.